<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsVectorGradientColorRampV2DialogBase</class>
 <widget class="QDialog" name="QgsVectorGradientColorRampV2DialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gradient color ramp</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0,0,0,1">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Color &amp;1</string>
       </property>
       <property name="buddy">
        <cstring>btnColor1</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QgsColorButtonV2" name="btnColor1">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Color &amp;2</string>
       </property>
       <property name="buddy">
        <cstring>btnColor2</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QgsColorButtonV2" name="btnColor2">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>&amp;Type</string>
       </property>
       <property name="buddy">
        <cstring>cboType</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cboType">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToMinimumContentsLength</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QgsGradientStopEditor" name="mStopEditor" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QgsCollapsibleGroupBox" name="groupBox_2">
     <property name="title">
      <string>Gradient stop</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" rowminimumheight="0,1">
      <item row="0" column="1">
       <widget class="QgsDoubleSpinBox" name="mPositionSpinBox">
        <property name="suffix">
         <string> %</string>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Relative &amp;position</string>
        </property>
        <property name="buddy">
         <cstring>mPositionSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="mDeleteStopButton">
        <property name="text">
         <string>&amp;Delete</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="4">
       <widget class="QgsCompoundColorWidget" name="mColorWidget" native="true">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnInformation">
       <property name="text">
        <string>&amp;Information</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsColorButtonV2</class>
   <extends>QToolButton</extends>
   <header>qgscolorbuttonv2.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsGradientStopEditor</class>
   <extends>QWidget</extends>
   <header>qgsgradientstopeditor.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qgsdoublespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsCompoundColorWidget</class>
   <extends>QWidget</extends>
   <header>qgscompoundcolorwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>btnColor1</tabstop>
  <tabstop>btnColor2</tabstop>
  <tabstop>cboType</tabstop>
  <tabstop>mStopEditor</tabstop>
  <tabstop>mPositionSpinBox</tabstop>
  <tabstop>mDeleteStopButton</tabstop>
  <tabstop>mColorWidget</tabstop>
  <tabstop>btnInformation</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QgsVectorGradientColorRampV2DialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>354</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QgsVectorGradientColorRampV2DialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>354</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
